Automattic/wp-calypso
# Site Assembler - Categories - Remove feature flag pattern-assembler/categories

Related to #72563

## Proposed Changes

* Remove the feature flag `pattern-assembler/categories`
* Remove the component `ScreenPatternList`

## Testing Instructions



**Everything should work as expected. Nothing is changing on the user's end.**

* Use the `Calypso live` link on the first comment
* Create a new site and continue until the Design Picker
* Click `Start designing` from the bottom of the design picker 
* Choose a header, then `Save`
* Choose a footer, then `Save`
* Click `Sections` 
* Click `Add patterns` to explore the categories and add patternst
* Replace, order, and delete patterns
* Click `Continue` and verify the patterns are added to a Home template on your site

## Pre-merge Checklist
